<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FnWebInfoPage</class>
 <widget class="QWidget" name="FnWebInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="edtTitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtUrl">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtPage" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
